### PR TITLE
Improve installation

### PR DESCRIPTION
**Pull request by krtk6160**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -18,7 +18,18 @@ src/tool/users_balance.csv
 src/tool/record_transactions_*.csv
 
 *.DS_Store
-configs
+charts/configs/*
+!charts/configs/lnd
+charts/configs/lnd/*
+!charts/configs/lnd/regtest.yaml
```
Comment (nicolasburtey) [inline .gitignore:24]: those files are not commited in this PR?

Comment (krtk6160) [reply .gitignore:24]: They are

File: .gitignore [hunk 2]
```diff
@@ -18,7 +18,18 @@ src/tool/users_balance.csv
...
+!charts/configs/lnd/regtest-outside.yaml
+!charts/configs/lnd/localdev.yaml
+!charts/configs/lnd/localdev-outside.yaml
+
+!charts/configs/bitcoind
+charts/configs/bitcoind/*
+!charts/configs/bitcoind/regtest.yaml
+!charts/configs/bitcoind/localdev.yaml
```
Comment (nicolasburtey) [inline .gitignore:32]: I feel those config file could be in a different folder but maybe this is for another PR

File: charts/configs/bitcoind/localdev.yaml [hunk 3]
```diff
@@ -0,0 +1,2 @@
+service:
```
Comment (nicolasburtey) [inline charts/configs/bitcoind/localdev.yaml:1]: maybe add a comment why we need that (necessary for minikube to be able to access from outside the cluster?)

**Review by nicolasburtey — APPROVED** [review]
just one comment but good to go for me

File: .gitignore (hunk 2 above)
Comment (nicolasburtey) [reply .gitignore:32]: those files are also duplicated in the config repo. probably should be deleted there to avoid confusion.

Comment (krtk6160) [reply .gitignore:32]: I was planning on keeping a set in configs as well to allow us to overwrite these files without having to commit them to the public galoy repo. On second thought maybe it's something we wouldn't require, so I'll remove it.

**Pull request merged**